tendrl/commons/objects/cluster_network_details/__init__.py

```diff
@@ -0,0 +1,17 @@
+from tendrl.commons import objects
+
+
+class ClusterNetworkDetails(objects.BaseObject):
```

Comment:
There is already a "Cluster" object, implement it here https://github.com/Tendrl/commons/blob/develop/tendrl/commons/objects/definition/master.yaml#L85